### ISSUE TITLE
No xz executable found, using decompress()

### ISSUE THREAD

**Issue by humboldtux** [grahamgilbert/imagr#161]
I'm getting this warning multiple times at the end of the generation.

Is it important? How can i install xz executable?

Sorry if it's to AutoNBI.

Version: Imagr.1.3.3

**Comment by grahamgilbert** [comment]
This is related to [AutoNBI](https://bitbucket.org/bruienne/autonbi) rather than Imagr. I do not believe it is important.

**Comment by ivydigitalstorm** [comment]
hey, @humboldtux cc @grahamgilbert, i know this is closed, but in case anyone else is having issues with xz I have a small finding related to this error.

I observed the same warnings, although 'make nbi'  would complete successfully, albeit slowly. However, OS X Server NetInstall service wouldn't detect the resulting Imagr.nbi files i placed in /Library/NetBoot/NetBootSP0/ 

i used 'brew install xz' and ran 'make nbi' again and the resulting files were then recognized by NetInstall service.  May be a coincidence, but I found it was worthwhile to remedy the complaint in order to get NetInstall to recognize Imagr.nbi  (even if it is more related to AutoNBI than Imagr)
